cdflow_commands/deploy.py

```diff
@@ -0,0 +1,107 @@
+import json
+import os
+from os import path
+from subprocess import check_call
+from tempfile import NamedTemporaryFile
+from time import time
+
+from cdflow_commands.constants import (
+    CONFIG_BASE_PATH, GLOBAL_CONFIG_FILE, INFRASTRUCTURE_DEFINITIONS_PATH,
+    PLATFORM_CONFIG_BASE_PATH, RELEASE_METADATA_FILE, TERRAFORM_BINARY
+)
+from cdflow_commands.secrets import get_secrets
+
+
+class Deploy:
+
+    def __init__(
+        self, environment, release_path, account_scheme, boto_session,
+    ):
+        self._environment = environment
+        self._release_path = release_path
+        self._account_scheme = account_scheme
+        self._boto_session = boto_session
+
+    def run(self, plan_only=False):
+        self._plan()
+        if not plan_only:
+            self._apply()
+
+    def _plan(self):
+        with NamedTemporaryFile() as secrets_file_path:
+            json.dump(get_secrets(), secrets_file_path)
+            check_call(
+                self._build_parameters('plan', secrets_file_path),
```

Comment:
No, that's bundled into the release: https://github.com/mergermarket/cdflow-commands/blob/c1cac077f111c4dff96aa05e02f43dce64b3efd0/cdflow_commands/release.py#L83

I didn't want to bundle decrypted config values though.